compiler/damlc/tests/daml-test-files/Unicode.daml

```diff
@@ -2,9 +2,9 @@
 -- All rights reserved.
 
 -- @ERROR range=9:1-9:5; ⛄ ¯\_(ツ)_/¯
-
-
 module Unicode where
 
-main = scenario do
+import Daml.Script
+
+main = script @() do
```

Comment:
Given the amount of times where we need type application for script with @(), do you reckon it's worth adding when this is needed to the script introduction (https://docs.daml.com/daml-script/index.html)?

Reply:
Unsure, its only really needed when you're using errors to track what point you got to, which I'm not entirely convinced is even a reasonable pattern...
It couldn't hurt though, maybe specifically under wherever `daml test` is documented, as that'll mention it runs top level scripts

Reply:
Do note that the warning for this does exist, I'd be told if I forgot that `@()`

Reply:
Filed #16584. I think we can leave it for now.